src/components/policy/policy_external/src/sql_pt_representation.cc

```diff
@@ -1033,6 +1074,22 @@ bool SQLPTRepresentation::SaveSpecificAppPolicy(
   if (!SaveAppGroup(app.first, app.second.groups)) {
     return false;
   }
+
+#ifdef SDL_REMOTE_CONTROL
+  if (!SaveAppGroupPrimary(app.first, *app.second.groups_primaryRC)) {
```

Comment:
in general, gorups_primary , groups_nonPrimary, xxx Consent_PassengerRC is not need by RC baseline, please remove them